app/models/student_task.rb

```
@@ -1,50 +1,173 @@
+    in_work_stage? && !revise?
+  end
+
+  private
+
+  def in_work_stage?
+    [ExpertizaConstants::DeadlineTypes::NAMES[ExpertizaConstants::DeadlineTypes::SUBMISSION],
+     ExpertizaConstants::DeadlineTypes::NAMES[ExpertizaConstants::DeadlineTypes::REVIEW]].include?(current_stage)
+  end
+
+  def content_submitted_in_current_stage?
+    return false unless current_stage == 'submission'
```

RuboCop (1.87.0)

[convention] Add empty line after guard clause.

(Layout/EmptyLineAfterGuardClause)